package.json

```diff
@@ -8,6 +8,7 @@
     "react-dom": ">=0.14.0 || ^0.14.0-beta3"
   },
   "devDependencies": {
```

Comment:
Does it make sense for example dependencies to be tracked as devDependencies?  Should React be tracked as a devDependency as well?